transport/http/handler.go

```diff
@@ -82,6 +83,11 @@ func (h handler) callHandler(w http.ResponseWriter, req *http.Request) error {
 
 	ctx, cancel := context.WithTimeout(context.Background(), treq.TTL)
 	defer cancel()
+
+	if hs := baggageHeaders.FromHTTPHeaders(req.Header, nil); len(hs) > 0 {
```

Comment:
If it's just a temp var for this block, then `h` might be better, else suggest the full `headers` here.